lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/InteropLogger.java

```diff
@@ -41,15 +41,22 @@
     levelHashMap.put(Level.ALL.intValue(), Level.ALL);
   }
 
-  public void Log(int intLevel, String message) {
```

Comment:
Looks like this method is used in cpp code too 
F:\incubator-reef\lang\cs\Org.Apache.REEF.Bridge\InteropLogger.cpp
{{_jmidLog = env->GetMethodID(_jclassInteropLogger, "Log", "(ILjava/lang/String;)V");}}
So we should rename it there too instead of deprecation path, same as for other bridge methods, right?